jbpm-db-scripts/src/main/resources/db/ddl-scripts/sybase/task_assigning_tables_sybase.sql

```diff
@@ -0,0 +1,15 @@
+    create table PlanningTask (
+        taskId numeric(19,0) not null,
+        assignedUser varchar(255),
+        taskIndex int not null,
+        lastModificationDate datetime,
+        published smallint not null,
+        primary key (taskId)
+    )
```

Comment:
The jBPM sql script uses "lock datarows" for every table it creates. Is there any motivation for keeping the default "page locking" of sybase [1]?

[1] http://infocenter.sybase.com/help/index.jsp?topic=/com.sybase.dc20021_1251/html/locking/locking9.htm

Reply:
good catch, no there's no reason. Just forgot to add it, I'll fix it.